src/ifp/test/mock.def

```diff
@@ -5,7 +5,7 @@ DESIGN simple_design ;
 UNITS DISTANCE MICRONS 1000 ;
 DIEAREA ( 0 0 ) ( 2000 2000 ) ;
 ROW ROW_0 SingleHeightSite 0 0 FS DO 10 BY 1 STEP 1000 0 ;
-ROW ROW_1 DoubleHeightSite 1200 2200 N DO 12 BY 1 STEP 1100 0 ;
+ROW ROW_1 DoubleHeightSite 0 1000 N DO 10 BY 1 STEP 1000 0 ;
```

Comment:
Why are there rows in the input DEF?  The goal of ifp is to build the rows.  You could just read a .v for this test which would better mirror ORFS (ie the yosys output).

Reply:
makes sense. I will fix it